src/app/core/pages/manage/manage.component.ts

```diff
@@ -177,4 +183,25 @@ export class ManageComponent implements OnInit {
         }
         this.sortTypeInternships = type;
     }
+
+    // Delete functions
+    deleteAct(activity) {
```

Comment:
Why is only Activity abbreviated? Especially as internship is longer

Reply:
fair point. Probably because i didnt care enough xD